### PR TITLE
Lm/detail view

### PR DESCRIPTION
**Pull request by levimllr**
Got the basics of a detail card view down. Let me know what you think of it so far!

### CODE REVIEW

File: src/DetailView/DetailView.tsx [hunk 1]
```diff
@@ -1,19 +1,139 @@
...
+
+const typeHeader = (types: Array<Type>) => {
+  return types.map(type => (
+    <div className={`type-header-${type.type.name} view-header`}>
+      {type.type.name}
+    </div>
+  ));
+};
+
+const heightConverter = (height: number) => {
```
Comment (qualitydixon) [inline src/DetailView/DetailView.tsx]: Some of these are good candidates to move into a utils or helpers file. Make a `src/utils.ts` or `src/helpers.ts` (dealer's choice).

File: src/DetailView/DetailView.tsx [hunk 2]
```diff
@@ -1,19 +1,139 @@
...
+  const inches = totalInches % 12;
+  let heightString = '';
+  if (feet > 0) {
+    heightString += `${feet}' `;
+  }
+  heightString += `${inches}"`;
+  return heightString;
+};
+
+const weightConverter = (weight: number) => {
```
Comment (qualitydixon) [inline src/DetailView/DetailView.tsx]: util

File: src/DetailView/DetailView.tsx [hunk 3]
```diff
@@ -1,19 +1,139 @@
...
+  return moveSample.map(move => (
+    <tr>
+      <td className="align-cell-left">
+        {capitalizeFirstLetter(move.move.name)}
+      </td>
+    </tr>
+  ));
+};
+
+const capitalizeFirstLetter = (string: string) => {
```
Comment (qualitydixon) [inline src/DetailView/DetailView.tsx]: util

File: src/DetailView/DetailView.tsx [hunk 4]
```diff
@@ -1,19 +1,139 @@
...
+    <tr>
+      <td className="align-cell-right">
+        {capitalizeFirstLetter(stat.stat.name)}
+      </td>
+      <td className="align-cell-right">{stat.base_stat}</td>
+    </tr>
+  ));
+};
+
+const moveFormatter = (moves: Array<Move>) => {
```
Comment (qualitydixon) [inline src/DetailView/DetailView.tsx]: Maybe a comment/doc string for what this is doing? Looks like we're pulling out and displaying 5 random moves?

File: src/DetailView/DetailView.tsx (hunk 1 above)
Comment (levimllr) [reply src/DetailView/DetailView.tsx]: Good point! Done.

File: src/DetailView/DetailView.tsx (hunk 3 above)
Comment (levimllr) [reply src/DetailView/DetailView.tsx]: helpers

**Pull request merged**